streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -1029,7 +1033,13 @@ public String getMain(Application application) {
     @Override
     @RefreshCache
     public boolean mapping(Application appParam) {
-        boolean mapping = this.baseMapper.mapping(appParam);
+        appParam.setAppId(appParam.getAppId());
+        appParam.setJobId(appParam.getJobId());
+        appParam.setState(14);
```

Comment:
you can improve the code: `appParam.setState(FlinkAppState.MAPPING.getValue());`